.env.example

```
@@ -10,6 +10,7 @@ AZURE_SUBSCRIPTION_KEY=
 BASE_URL=http://localhost:3000
 GA_TRACKING_ID=
 IPFS_ENDPOINT=
+IS_MAINTENANCE_MODE=FALSE
```

```suggestion
IS_MAINTENANCE_MODE=
```
I think not setting env means `FALSE` wouldbe more robust
Otherwise we might accidentally get stuff like `True` `TRUE` `1` `False` `false` `0` etc